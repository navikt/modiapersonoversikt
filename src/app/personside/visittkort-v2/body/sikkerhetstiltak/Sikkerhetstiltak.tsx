import { Normaltekst } from 'nav-frontend-typografi';
import SikkerhetsTiltakIkon from '../../../../../svg/Sikkerhetstiltak';
import type { Sikkerhetstiltak as SikkerhetstiltakInterface } from '../../PersondataDomain';
import GyldighetsPeriode from '../GyldighetsPeriode';
import VisittkortElement from '../VisittkortElement';
import { VisittkortGruppe } from '../VisittkortStyles';

interface Props {
    sikkerhetstiltak: SikkerhetstiltakInterface[];
}

function Sikkerhetstiltak({ sikkerhetstiltak }: Props) {
    if (sikkerhetstiltak.isEmpty()) {
        return null;
    }

    return (
        <VisittkortGruppe tittel="Sikkerhetstiltak" ikon={<SikkerhetsTiltakIkon />}>
            <VisittkortElement>
                {sikkerhetstiltak.map((sikkerhetstiltak, index) => {
                    return (
                        <div key={index}>
                            <GyldighetsPeriode gyldighetsPeriode={sikkerhetstiltak.gyldighetsPeriode} />
                            <Normaltekst>{sikkerhetstiltak.beskrivelse}</Normaltekst>
                        </div>
                    );
                })}
            </VisittkortElement>
        </VisittkortGruppe>
    );
}

export default Sikkerhetstiltak;
