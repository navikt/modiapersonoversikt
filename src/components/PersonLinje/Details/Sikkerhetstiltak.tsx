import { ExclamationmarkTriangleFillIcon } from '@navikt/aksel-icons';
import { BodyShort } from '@navikt/ds-react';
import { usePersonData } from 'src/lib/clients/modiapersonoversikt-api';
import ValidPeriod from '../common/ValidPeriod';
import { Group, InfoElement } from './components';

function Sikkerhetstiltak() {
    const { data } = usePersonData();
    const { sikkerhetstiltak } = data.person;

    if (sikkerhetstiltak.isEmpty()) {
        return null;
    }

    return (
        <Group title="Sikkerhetstiltak" icon={<ExclamationmarkTriangleFillIcon color="var(--a-icon-warning)" />}>
            <InfoElement>
                {sikkerhetstiltak.map((sikkerhetstiltak) => {
                    return (
                        <div key={sikkerhetstiltak.beskrivelse}>
                            <ValidPeriod
                                from={sikkerhetstiltak.gyldighetsPeriode?.gyldigFraOgMed}
                                to={sikkerhetstiltak.gyldighetsPeriode?.gyldigTilOgMed}
                            />
                            <BodyShort>{sikkerhetstiltak.beskrivelse}</BodyShort>
                        </div>
                    );
                })}
            </InfoElement>
        </Group>
    );
}

export default Sikkerhetstiltak;
