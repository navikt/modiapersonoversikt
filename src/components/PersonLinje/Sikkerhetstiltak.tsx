import { ExclamationmarkTriangleFillIcon } from '@navikt/aksel-icons';
import { BodyShort, Button, Modal } from '@navikt/ds-react';
import { useEffect, useRef } from 'react';
import type { PersonData } from 'src/lib/types/modiapersonoversikt-api';
import ValidPeriod from './common/ValidPeriod';

type Sikkerhetstiltak = PersonData['sikkerhetstiltak'];

interface Props {
    sikkerhetstiltak: Sikkerhetstiltak;
}

export const Sikkerhetstiltak = ({ sikkerhetstiltak }: Props) => {
    const ref = useRef<HTMLDialogElement>(null);

    useEffect(() => {
        if (sikkerhetstiltak.isNotEmpty()) {
            ref.current?.showModal();
        }
    }, [sikkerhetstiltak]);

    return (
        <Modal
            ref={ref}
            header={{
                heading: 'Sikkerhetstiltak',
                icon: <ExclamationmarkTriangleFillIcon color="var(--a-icon-danger)" />,
                closeButton: false
            }}
        >
            <Modal.Body>
                <ul className="p-4">
                    {sikkerhetstiltak.map((s) => (
                        <li className="my-4" key={s.beskrivelse}>
                            <ValidPeriod
                                from={s.gyldighetsPeriode?.gyldigFraOgMed}
                                to={s.gyldighetsPeriode?.gyldigTilOgMed}
                            />
                            <BodyShort>{s.beskrivelse}</BodyShort>
                        </li>
                    ))}
                </ul>
            </Modal.Body>
            <Modal.Footer>
                <Button onClick={() => ref.current?.close()}>Ok</Button>
            </Modal.Footer>
        </Modal>
    );
};
