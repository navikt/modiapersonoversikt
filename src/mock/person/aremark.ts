import { Diskresjonskoder, TilrettelagtKommunikasjonsTyper } from '../../konstanter';
import { Kjønn, Person, Relasjonstype, SivilstandTyper } from '../../models/person/person';

export const aremark: Person = {
    fødselsnummer: '10108000398',
    kjønn: Kjønn.Mann,
    geografiskTilknytning: '0118',
    alder: 42,
    navn: {
        sammensatt: 'AREMARK TESTFAMILIEN',
        fornavn: 'AREMARK',
        mellomnavn: '',
        etternavn: 'TESTFAMILIEN',
    },
    tilrettelagtKomunikasjonsListe: [
        {
            behovKode: 'TOHJ',
            beskrivelse: TilrettelagtKommunikasjonsTyper.TOHJ
        }
    ],
    diskresjonskode: Diskresjonskoder.FORTROLIG_ADRESSE,
    statsborgerskap: 'NORSK',
    personstatus: {
        dødsdato: undefined,
        bostatus: undefined
    },
    sivilstand: {
        value: SivilstandTyper.Gift,
        beskrivelse: 'Gift',
        fraOgMed: '2005-12-12'
    },
    familierelasjoner: [{
        harSammeBosted: false,
        rolle: Relasjonstype.Barn,
        tilPerson: {
            navn: {
                sammensatt: '',
                fornavn: 'Siri',
                mellomnavn: null,
                etternavn: 'Aremark'
            },
            alder: 17,
            fødselsnummer: '10117100000',
            personstatus: {}
        }

    }],
    kontaktinformasjon: {
        mobil: {
            sistEndret: '2014-06-21T18:44:39+02:00',
            telefonnummer: '99887766',
            retningsnummer: '47',
            sistEndretAv: 'BRUKER'
        }
    }
};