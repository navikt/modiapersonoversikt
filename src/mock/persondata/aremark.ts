import {
    AdresseBeskyttelse,
    EgenAnsatt,
    ForelderBarnRelasjon,
    ForelderBarnRelasjonRolle,
    FullmaktsRolle,
    Kjonn,
    LocalDate,
    LocalDateTime,
    Person,
    PersonStatus,
    SivilstandType,
    Skifteform
} from '../../app/personside/visittkort-v2/PersondataDomain';
import { harDiskresjonskode } from '../../app/personside/visittkort-v2/visittkort-utils';

export function hentBarnAremark(): ForelderBarnRelasjon[] {
    return [
        {
            ident: '12345678910',
            rolle: ForelderBarnRelasjonRolle.BARN,
            navn: [
                {
                    fornavn: 'BARN1',
                    mellomnavn: null,
                    etternavn: 'BARNESEN'
                }
            ],
            fodselsdato: ['2008-03-15' as LocalDate],
            alder: 13,
            kjonn: [
                {
                    kode: Kjonn.K,
                    beskrivelse: 'Kvinne'
                }
            ],
            adressebeskyttelse: [
                {
                    kode: AdresseBeskyttelse.UGRADERT,
                    beskrivelse: 'UGRADERT'
                }
            ],
            harSammeAdresse: false,
            dodsdato: []
        },
        {
            ident: '12345678911',
            rolle: ForelderBarnRelasjonRolle.BARN,
            navn: [
                {
                    fornavn: 'BARN1',
                    mellomnavn: null,
                    etternavn: 'BARNESEN'
                }
            ],
            fodselsdato: ['2008-03-15' as LocalDate],
            alder: null,
            kjonn: [
                {
                    kode: Kjonn.K,
                    beskrivelse: 'Kvinne'
                }
            ],
            adressebeskyttelse: [],
            harSammeAdresse: false,
            dodsdato: []
        }
    ];
}

export const aremark: Person = {
    fnr: '10108000398',
    navn: [
        {
            fornavn: 'AREMARK',
            mellomnavn: null,
            etternavn: 'TESTFAMILIEN'
        }
    ],
    kjonn: [
        {
            kode: Kjonn.M,
            beskrivelse: 'Mann'
        }
    ],
    fodselsdato: ['1980-10-10' as LocalDate],
    geografiskTilknytning: '0118',
    alder: 41,
    dodsdato: [],
    bostedAdresse: [
        {
            linje1: 'Islandsgate 49',
            linje2: '7693 Hovedøya',
            linje3: 'Norge',
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2010-02-20T10:15:30' as LocalDateTime,
                system: 'FREG'
            },
            gyldighetsPeriode: null
        }
    ],
    kontaktAdresse: [
        {
            linje1: 'Aremarkveien 1',
            linje2: '0320 Aremark',
            linje3: null,
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2020-01-01T10:15:30' as LocalDateTime,
                system: 'FREG'
            },
            gyldighetsPeriode: null
        }
    ],
    oppholdsAdresse: [
        {
            linje1: 'Oppholdsadresse 1',
            linje2: '0000 AREMARK',
            linje3: null,
            sistEndret: null,
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    navEnhet: {
        id: '0219',
        navn: 'NAV Bærum',
        publikumsmottak: [
            {
                besoksadresse: {
                    linje1: 'Adressevei 1',
                    linje2: '0000 AREMARK',
                    linje3: null,
                    sistEndret: null,
                    gyldighetsPeriode: null
                },
                apningstider: [
                    {
                        ukedag: 'Mandag',
                        apningstid: '09.00 - 15.00'
                    },
                    {
                        ukedag: 'Tirsdag',
                        apningstid: '09.00 - 15.00'
                    },
                    {
                        ukedag: 'Onsdag',
                        apningstid: '10.00 - 16.00'
                    }
                ]
            }
        ]
    },
    statsborgerskap: [
        {
            land: {
                kode: 'NOR',
                beskrivelse: 'NORGE'
            },
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    adressebeskyttelse: [
        {
            kode: AdresseBeskyttelse.KODE6,
            beskrivelse: 'Kode 6'
        }
    ],
    sikkerhetstiltak: [
        {
            type: 'TFUS',
            beskrivelse: 'Telefonisk utestengelse',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    erEgenAnsatt: EgenAnsatt.JA,
    personstatus: [
        {
            kode: PersonStatus.BOSATT,
            beskrivelse: 'Bosatt i Norge'
        }
    ],
    sivilstand: [
        {
            type: {
                kode: SivilstandType.GIFT,
                beskrivelse: 'Gift'
            },
            gyldigFraOgMed: '2005-12-12' as LocalDate,
            sivilstandRelasjon: {
                fnr: '12345555555',
                navn: [
                    {
                        fornavn: 'BAREMARK',
                        mellomnavn: null,
                        etternavn: 'TESTFAMILIEN'
                    }
                ],
                alder: 40,
                adressebeskyttelse: [
                    {
                        kode: AdresseBeskyttelse.UGRADERT,
                        beskrivelse: 'UGRADERT'
                    }
                ],
                harSammeAdresse: true
            }
        }
    ],
    foreldreansvar: hentBarnAremark().map((barn) => ({
        ansvar: 'felles',
        ansvarlig: null,
        ansvarsubject: {
            navn: harDiskresjonskode(barn.adressebeskyttelse) ? null : barn.navn.firstOrNull(),
            ident: barn.ident
        }
    })),
    deltBosted: [
        {
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            },
            adresse: {
                linje1: `Adresseveien 1`,
                linje2: '0000 Aremark',
                linje3: null,
                sistEndret: null,
                gyldighetsPeriode: null
            }
        }
    ],
    dodsbo: [
        {
            adressat: {
                advokatSomAdressat: {
                    kontaktperson: {
                        fornavn: 'Advokat',
                        mellomnavn: null,
                        etternavn: 'Navnesen'
                    },
                    organisasjonsnavn: 'Advokatkontoret Aremark',
                    organisasjonsnummer: '01234567891011'
                },
                organisasjonSomAdressat: null,
                personSomAdressat: null
            },
            adresse: {
                linje1: 'Elgelia 20, 0000 Aremark',
                linje2: null,
                linje3: null,
                sistEndret: null,
                gyldighetsPeriode: null
            },
            registrert: '2010-02-02' as LocalDate,
            skifteform: Skifteform.OFFENTLIG,
            sistEndret: {
                ident: 'Folkeregisteret',
                tidspunkt: '2020-01-01T10:15:30' as LocalDateTime,
                system: 'Folkeregisteret'
            }
        }
    ],
    fullmakt: [
        {
            motpartsPersonident: '123456789',
            motpartsPersonNavn: {
                fornavn: 'Navn',
                mellomnavn: null,
                etternavn: 'Navnesen'
            },
            motpartsRolle: FullmaktsRolle.FULLMEKTIG,
            omrade: [
                {
                    kode: '*',
                    beskrivelse: ''
                }
            ],
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-06-01' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    vergemal: [
        {
            ident: '21042900076',
            navn: {
                fornavn: 'Simen',
                mellomnavn: null,
                etternavn: 'Solli'
            },
            vergesakstype: 'Voksen',
            omfang: 'Ivareta personens interesser innenfor det personlige og økonomiske området',
            embete: 'Fylkesmannen i Troms og Finnmark',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-06-01' as LocalDate,
                gyldigTilOgMed: '2022-05-10' as LocalDate
            }
        },
        {
            ident: '123456799',
            navn: {
                fornavn: 'Truls',
                mellomnavn: null,
                etternavn: 'Tøffel'
            },
            vergesakstype: 'Fremtidsfullmakt',
            omfang: 'Ivareta personens interesser innenfor det økonomiske området',
            embete: 'Fylkesmannen i Troms og Finnmark',
            gyldighetsPeriode: {
                gyldigFraOgMed: '2021-02-10' as LocalDate,
                gyldigTilOgMed: '2022-05-01' as LocalDate
            }
        }
    ],
    tilrettelagtKommunikasjon: {
        talesprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ],
        tegnsprak: [
            {
                kode: 'NO',
                beskrivelse: 'Norsk'
            }
        ]
    },
    telefonnummer: [
        {
            retningsnummer: {
                kode: '+47',
                beskrivelse: 'Norge'
            },
            identifikator: '99009900',
            sistEndret: {
                ident: '11223344',
                tidspunkt: '2018-06-14T00:00:00' as LocalDateTime,
                system: 'BD06'
            },
            prioritet: -1
        }
    ],
    kontaktOgReservasjon: {
        personident: '10108000398',
        reservasjon: 'true',
        epostadresse: {
            value: 'epost@nav.no',
            sistOppdatert: null,
            sistVerifisert: '2013-01-01' as LocalDate
        },
        mobiltelefonnummer: {
            value: '12345678',
            sistOppdatert: '2021-11-02' as LocalDate,
            sistVerifisert: '2021-11-02' as LocalDate
        }
    },
    bankkonto: {
        kontonummer: '12345678900',
        banknavn: 'DNB ASA',
        sistEndret: {
            ident: '1010800 BD03',
            tidspunkt: '2006-03-15T00:00:00' as LocalDateTime,
            system: ''
        },
        bankkode: null,
        swift: 'DNBANOKKXXX',
        landkode: null,
        adresse: {
            linje1: 'Bankveien 1,',
            linje2: '0357 Bankestad',
            linje3: null,
            sistEndret: null,
            gyldighetsPeriode: null
        },
        valuta: {
            kode: 'NOK',
            beskrivelse: 'Norske kroner'
        }
    },
    forelderBarnRelasjon: hentBarnAremark()
};
