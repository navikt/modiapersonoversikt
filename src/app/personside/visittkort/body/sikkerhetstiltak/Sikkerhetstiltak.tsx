import * as React from 'react';
import { Sikkerhetstiltak as SikkerhetstiltakInterface } from '../../../../../models/sikkerhetstiltak';
import { Person } from '../../../../../models/person/person';
import { Normaltekst } from 'nav-frontend-typografi';
import { Periode } from '../../../../../models/periode';
import VisPeriode from '../../../../../components/person/VisPeriode';
import SikkerhetsTiltakIkon from '../../../../../svg/Sikkerhetstiltak';
import { VisittkortGruppe } from '../VisittkortStyles';
import VisittkortElement from '../VisittkortElement';

interface SikkerhetstiltakProps {
    person: Person;
}

function Sikkerhetstiltak({ person }: SikkerhetstiltakProps) {
    if (person.sikkerhetstiltak == null) {
        return null;
    }
    return (
        <VisittkortGruppe tittel="Sikkerhetstiltak" ikon={<SikkerhetsTiltakIkon />}>
            {tiltaksinfo(person.sikkerhetstiltak)}
        </VisittkortGruppe>
    );
}

function tiltaksinfo(sikkerhetstiltak: SikkerhetstiltakInterface) {
    return (
        <VisittkortElement>
            {hentPeriode(sikkerhetstiltak.periode)}
            <Normaltekst>{sikkerhetstiltak.sikkerhetstiltaksbeskrivelse}</Normaltekst>
        </VisittkortElement>
    );
}

function hentPeriode(periode?: Periode) {
    if (periode != null) {
        return <VisPeriode periode={periode} />;
    }
    return null;
}

export default Sikkerhetstiltak;
