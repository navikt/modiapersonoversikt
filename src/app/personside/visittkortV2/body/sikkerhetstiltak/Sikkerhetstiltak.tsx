import * as React from 'react';
import { Person } from '../../../../../models/personPdl/person';
import { Sikkerhetstiltak } from '../../../../../models/personPdl/sikkerhetstiltak';
import { VisittkortGruppe } from '../VisittkortStyles';
import SikkerhetsTiltakIkon from '../../../../../svg/Sikkerhetstiltak';
import VisittkortElement from '../VisittkortElement';
import { Normaltekst } from 'nav-frontend-typografi';

interface SikkerhetstiltakProps {
    person: Person;
}

function Sikkerhetstiltak({ person }: SikkerhetstiltakProps) {
    if (person.sikkerhetstiltak.length === 0) {
        return null;
    }

    const sikkerhetstiltakListe = person.sikkerhetstiltak.map(sikkerhetstiltak => {
        Tiltaksinfo(sikkerhetstiltak);
    });

    return (
        <VisittkortGruppe tittel="Sikkerhetstiltak" ikon={<SikkerhetsTiltakIkon />}>
            {sikkerhetstiltakListe}
        </VisittkortGruppe>
    );
}

function Tiltaksinfo(sikkerhetstiltak: Sikkerhetstiltak) {
    return (
        <VisittkortElement>
            `${sikkerhetstiltak.gyldigFraOgMed} ${sikkerhetstiltak.gyldigTilOgMed}`
            <Normaltekst>{sikkerhetstiltak.type}</Normaltekst>
        </VisittkortElement>
    );
}

export default Sikkerhetstiltak;
