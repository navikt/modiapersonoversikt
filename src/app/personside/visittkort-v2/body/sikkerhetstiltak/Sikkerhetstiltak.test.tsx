import * as renderer from 'react-test-renderer';
import { aremark } from '../../../../../mock/persondata/aremark';
import TestProvider from '../../../../../test/Testprovider';
import Sikkerhetstiltak from './Sikkerhetstiltak';

test('viser sikkerhetstiltak', () => {
    const visittkortbody = renderer.create(
        <TestProvider>
            <Sikkerhetstiltak sikkerhetstiltak={aremark.sikkerhetstiltak} />
        </TestProvider>
    );

    expect(visittkortbody.toJSON()).toMatchSnapshot();
});
