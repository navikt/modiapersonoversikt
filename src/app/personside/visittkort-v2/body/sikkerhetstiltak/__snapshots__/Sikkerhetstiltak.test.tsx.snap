// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`viser sikkerhetstiltak 1`] = `
.c0 {
  position: relative;
  display: flex;
  align-items: center;
}

.c1 {
  position: absolute;
  left: -3.125rem;
  width: 3.125rem;
  display: flex;
  justify-content: center;
}

.c1 >svg {
  height: 1.5rem;
  width: auto;
}

.c2 {
  margin-bottom: 1rem;
}

.c3 {
  margin: 10px 0 20px 0;
}

.c3:last-child {
  margin: 10px 0 0 0;
}

.c4 {
  color: #645f5a;
  margin-top: 3px;
}

.c4 .typo-etikett-liten {
  line-height: 1rem;
}

<section>
  <div
    className="c0"
  >
    <div
      className="c1"
    >
      <svg
        height={100}
        viewBox="0 0 100 100"
        width={100}
        xmlns="http://www.w3.org/2000/svg"
      >
        <defs>
          <mask
            id="sikkerhetstiltakmask"
          >
            <rect
              fill="#fff"
              height="100%"
              width="100%"
            />
            <line
              stroke="#000"
              strokeLinecap="round"
              strokeWidth="4"
              x1="50"
              x2="50"
              y1="35"
              y2="67"
            />
            <circle
              cx="50"
              cy="82"
              fill="#000"
              r="4.5"
            />
          </mask>
        </defs>
        <polygon
          fill="#BA3A26"
          mask="url(#sikkerhetstiltakmask)"
          points="50,5 95,95 5,95"
          stroke="#BA3A26"
          strokeLinejoin="round"
          strokeWidth="4"
        />
      </svg>
    </div>
    <h3
      className="typo-undertittel"
    >
      Sikkerhetstiltak
    </h3>
  </div>
  <div
    className="c2"
  />
  <div
    className="c3"
  >
    <div
      className="c0"
    />
    <div>
      <div
        className="c4"
      >
        <p
          className="typo-undertekst"
        >
          Gyldig: 
          01.02.2021 â€“ 01.05.2022
        </p>
      </div>
      <p
        className="typo-normal"
      >
        Telefonisk utestengelse
      </p>
    </div>
  </div>
</section>
`;
